src/material/datepicker/datepicker.md

```diff
@@ -588,6 +588,8 @@ attribute to the native input and button elements, respectively.
 the datepicker text input a meaningful label via `<mat-label>`, `aria-label`, `aria-labelledby` or
 `MatDatepickerIntl`.
 
+Always communicate the date format (e.g. 'MM/DD/YYYY'). This can be accomplished using `<mat-hint>`.
```

Comment:
```suggestion
Always communicate the date format (e.g. 'MM/DD/YYYY'). This can be accomplished using `<mat-hint>` or by providing an additional label adjacent to the form field.
```

Reply:
done ✔️